### PR TITLE
Sentry passing props dispatch to ItemParent

### PR DESCRIPTION
**Pull request by d88naimi**
Sentry issue https://sentry.io/organizations/zestyio/issues/2328692196/?project=5441698

### CODE REVIEW

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const ItemSettings = memo(
                 path={web.path}
                 onChange={onChange}
                 currentItemLangID={props.item.meta.langID}
+                dispatch={props.dispatch}
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js:56]: This seems unrelated to the error message on the sentry bug referenced; 

> Cannot read properties of undefined (reading 'data')

That appears to be that the response is empty but we are still attempting to read the `data` property

**Review by shrunyan — CHANGES_REQUESTED** [review]

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js (hunk 1 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js:56]: Added Optional Chaining to `data`, but leaving the props.dispatch as ItemParent is is missing dispatch.

[the remark this answers is not in the document]
Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js:56]: Removed, but im not sure how props.dispatch is being used here https://github.com/zesty-io/manager-ui/blob/master/src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/ItemParent.js#L95

File: src/shell/store/middleware/api.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ export const fetchResource =
       if (inflight.indexOf(action.uri) === -1) {
         inflight.push(action.uri);
         return request(action.uri)
-          .then(action.handler)
+          .then((res) => {
```
Comment (shrunyan) [inline src/shell/store/middleware/api.js:16]: Let's switch back to the direct function reference

**Review by shrunyan — CHANGES_REQUESTED** [review]
